exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,46 @@
+class SpaceAgeCase < ExerciseCase
+  SECONDS_PER_YEAR_ON_EARTH = 31557600
+
+  def workload
+    [
+      "age = SpaceAge.new(#{underscore_format(seconds)})",
+      indent(4, assertion),
```

Comment:
Sounds good; I can use the ```indent_lines``` method of ```ExerciseCases```, and add the integer underscore formatting method I'm using to the ```Underscore``` module.